src/Control/GeoJsonService.php

```diff
@@ -33,13 +46,14 @@ public function index($request)
 
         $propertyMap = $config['property_map'];
 
-        header('Access-Control-Allow-Origin: ' . $config['access_control_allow_origin']);
+
+        $this->headerAlterer->setHeader('Access-Control-Allow-Origin: ' . $config['access_control_allow_origin']);
 
         // The HTTP kernel keeps a copy of the response body, which
         // can exhaust the memory limit for large data sets. So we
         // opt out and flush the buffer after processing each feature.
         if (!($is_test = headers_sent())) {
-            header('Content-Type: application/geo+json');
+            $this->headerAlterer->setHeader('Content-Type: application/geo+json');
```

Comment:
PHPUnit fails when header('some string') is called, so had to inject a non breaking proxy, record, and check headers